app/module/versioned_ibc_module_test.go

```diff
@@ -188,6 +190,17 @@ func TestVersionedIBCModule(t *testing.T) {
 			},
 			expectedValue: types.NewResultAcknowledgement([]byte("next_ack")),
 		},
+		{
+			name:    "OnAcknowledgementPacket with supported version",
+			version: 3,
+			setupMocks: func(ctx sdk.Context) {
+				mockWrappedModule.EXPECT().OnAcknowledgementPacket(ctx, types.Packet{}, []byte{}, sdk.AccAddress{}).Return(nil)
+			},
+			method: func(ctx sdk.Context) (interface{}, error) {
+				return nil, versionedModule.OnAcknowledgementPacket(ctx, types.Packet{}, []byte{}, sdk.AccAddress{})
+			},
+			expectedValue: nil,
+		},
```

Comment:
it would be nice to refactor these tests to make it easier to add a version without copy pasting gobs of code